clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -375,3 +376,27 @@ class KubernetesSeLinuxOptions {
   String type
   String level
 }
+
+@AutoClone
+@Canonical
+//Base on model https://kubernetes.io/docs/resources-reference/v1.5/#podspec-v1
+//which will map to Kubernetes 1.5 API version
+class KubernetesPodSpecDescription {
+  Long activeDeadlineSeconds
+  List<KubernetesContainerDescription> containers
+  String dnsPolicy
+  Boolean hostIPC
+  Boolean hostNetwork
+  Boolean hostPID
+  String hostname
```

Comment:
My naming is pretty much by following Kubernetes API - https://kubernetes.io/docs/resources-reference/v1.5/#podspec-v1
For example, 
hostname
string 	Specifies the hostname of the Pod If not specified, the pod's hostname will be set to a system-defined value.